app/(protected)/(books)/[bookId]/components/mini-book-selector.tsx

```
@@ -0,0 +1,68 @@
+
+export const MiniBookSelector = async ({
+  params,
+}: {
+  params: { bookId: string };
+}) => {
+  const bookId = params.bookId;
+  const supabase = createClient();
+  const bookRepo = new BookRepository(supabase);
+  const books = await bookRepo.getBooks();
+
+  const selectedBookTitle = !!bookId
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)